e2e-tests/functions

```diff
@@ -348,12 +348,14 @@ deploy_operator() {
 		cat ${src_dir}/deploy/cw-operator.yaml \
 			| sed -e "s^image: .*^image: ${IMAGE}^" \
 			| yq w - 'spec.template.spec.containers.(name==percona-server-mongodb-operator).env.(name==DISABLE_TELEMETRY).value' '"true"' \
+			| yq w - 'spec.template.spec.containers.(name==percona-server-mongodb-operator).env.(name==LOG_LEVEL).value' 'DEBUG' \
 			| kubectl_bin apply -f -
 	else
 		apply_rbac rbac
 		cat ${src_dir}/deploy/operator.yaml \
 			| sed -e "s^image: .*^image: ${IMAGE}^" \
 			| yq w - 'spec.template.spec.containers.(name==percona-server-mongodb-operator).env.(name==DISABLE_TELEMETRY).value' '"true"' \
+			| yq w - 'spec.template.spec.containers.(name==percona-server-mongodb-operator).env.(name==LOG_LEVEL).value' 'DEBUG' \
 			| kubectl_bin apply -f -
```

Comment:
Could you add this option to default-cr/run, deploy_operator_gh method in functions
https://github.com/percona/percona-server-mongodb-operator/blob/main/e2e-tests/default-cr/run#L57-L60
https://github.com/percona/percona-server-mongodb-operator/blob/main/e2e-tests/functions#L379

Reply:
Hey @nmarukovich , not sure I quite get you here, why do we need them in here? Tomislav suggested to me to put it just in `functions/deploy_operator()`

Reply:
I just want to say that there two extra places where we deploy operator. Why not use it here? @tplavcic we need your help (=

Reply:
@inelpandzic @nmarukovich IIRC the idea was to use DEBUG level in `deploy_operator()` and leave the `default-cr` test to use INFO level just so that we have a case for the default option also and `deploy_operator_gh()` is used for upgrade so the same applies here.
So I think this is ok as it is.